Dockerfile

```diff
@@ -10,41 +10,43 @@ COPY ./ ./
 # and so that source changes don't invalidate our downloaded layer
 RUN go mod download
 
+# Install git
+RUN apt-get update && apt-get install -y git curl gpg
+
 # Build
-WORKDIR /workspace
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o /bin/iter8 main.go
-WORKDIR /workspace
+RUN make build
+RUN cp ./bin/iter8 /bin/iter8
 
-# Install kubectl
-RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
-RUN chmod 755 kubectl
-RUN cp kubectl /bin
+# # Install kubectl
+# RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+# RUN chmod 755 kubectl
+# RUN cp kubectl /bin
 
-# Install Helm 3
-RUN curl -fsSL -o helm-v3.5.0-linux-amd64.tar.gz https://get.helm.sh/helm-v3.5.0-linux-amd64.tar.gz
-RUN tar -zxvf helm-v3.5.0-linux-amd64.tar.gz
-RUN linux-amd64/helm version
+# # Install Helm 3
+# RUN curl -fsSL -o helm-v3.5.0-linux-amd64.tar.gz https://get.helm.sh/helm-v3.5.0-linux-amd64.tar.gz
+# RUN tar -zxvf helm-v3.5.0-linux-amd64.tar.gz
+# RUN linux-amd64/helm version
 
-# Install Kustomize v3
-RUN curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash
-RUN cp kustomize /bin
+# # Install Kustomize v3
+# RUN curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh" | bash
+# RUN cp kustomize /bin
 
-# Install yq
-RUN GO111MODULE=on GOBIN=/bin go get github.com/mikefarah/yq/v4
+# # Install yq
+# RUN GO111MODULE=on GOBIN=/bin go get github.com/mikefarah/yq/v4
```

Comment:
Why are we making these changes?  These were here to support the exec action.

Reply:
Good catch. As we introduce more capabilities in the charts, for example, notifications, we can reassess which of these binaries are actually required and put them back into the docker image.
***
For now, they are simply bloating the image, without actually contributing to the only two experiments we support through the charts today, namely `load-test-http` and `load-test-grpc`. Hence, the commenting out ...